src/streamlink/plugins/wwenetwork.py

```diff
@@ -117,17 +94,8 @@ def _get_media_info(self, content_id):
         info = self.request("GET", self.stream_url.format(id=content_id))
         return self.request("GET", info.get("playerUrlCallback"))
 
-    def _get_video_id(self):
-        #  check the page to find the contentId
-        log.debug("Searching for content ID")
-        try:
-            if self.item_config["type"] == "channel":
-                return self._get_live_id()
-            else:
-                return "vod/{id}".format(id=self.item_config["customFields"]["DiceVideoId"])
-        except KeyError:
-            log.error("Could not find video ID")
-            return
+    def _get_video_id(self, video_id):
+        return "vod/{0}".format(video_id)
```

Comment:
This method could be removed, but it's not important. The plugin hasn't been updated in a while and in order to get its code in proper up2date shape, more changes would be required, so just ignore this.